cli/src/base.ts

```diff
@@ -100,10 +100,18 @@ export abstract class BaseCommand extends Command {
     };
   }
 
+  loadEnvFile(path: string) {
+    const env = dotenv.config({ path });
+    if (env.parsed?.['XATA_API_KEY']) {
+      this.apiKeyLocation = 'dotenv';
+      this.apiKeyDotenvLocation = path;
+    }
+  }
+
   async init() {
     if (process.env.XATA_API_KEY) this.apiKeyLocation = 'shell';
-    const env = dotenv.config({ path: this.dotenvLocation });
-    if (env.parsed?.['XATA_API_KEY']) this.apiKeyLocation = 'dotenv';
+    this.loadEnvFile('.env.local');
+    this.loadEnvFile('.env');
```

Comment:
Doesn't this overwrite the location on subsequent calls?

So, if user has `.env.local` and `.env`, it will pass all truthy conditional branches and write twice the variable, first with local and then with the normal one.

Similarly to the write below, I would pass an array and early return once first found.

Reply:
Yes, it overwrites the location, but that's intentional. If you have a `XATA_API_KEY` in both `.env.local` and `.env`, then the one in `.env` overwrites the one in `.env.local`.

I can do also an array, but the behavior won't change.

Reply:
Shouldn't it be the opposite?

If I have it in my local one, that's the one that should be used, not the one in normal dotenv 🤔

Reply:
🤔 I was following the next.js docs. After re-reading them I think you are right 👍